src/api.ts

```diff
@@ -44,10 +44,7 @@ export class API3 {
             name: id
         }
         try {
-            const response = await fetch(`https://${root}/${id}`, {
-                method: "GET",
-                cache: "force-cache"
```

Comment:
Just one comment/thing to check, I would be very cautious with reducing the cache amount. If the removal of `cache: "force-cache"` is causing more requests, this might have a negative impact on the overall system health as quite a few requests per page load are done.